src/consumer/kafka.rs

```diff
@@ -42,11 +44,20 @@ use tracing::{debug, error, info, instrument};
 pub async fn start_consumer(
     topics: &[&str],
     kafka_client_config: &ClientConfig,
+    default_topic_partitions: i32,
     spawn_actors: impl FnMut(
         Arc<StreamConsumer<KafkaContext>>,
         &BTreeSet<(String, i32)>,
     ) -> ActorHandles,
 ) -> Result<(), Error> {
+    create_missing_topics(
+        kafka_client_config.clone(),
+        topics,
+        default_topic_partitions,
+    )
+    .await
```

Comment:
We should call this function in `main`, before any of the green threads are spawned, so any workers do not try and make grpc connections.